### ISSUE TITLE
Deactivation: Reduce/eliminate manual steps

### ISSUE THREAD

**Issue by kraftbj** [Automattic/wp-super-cache#3]
As much as possible, let's reduce the manual steps needed per https://wordpress.org/plugins/wp-super-cache/other_notes/#How-to-uninstall-WP-Super-Cache

Related: #2

**Comment by liquidsunshine** [comment]
Oy.  This one bit me today.  I think this is the first plugin I ever had to go through and manually clean up after when removing.  (And the first one that remained active after deactivating.)

**Comment by kraftbj** [comment]
Agreed it is a pain point. I'll review what other solutions put in place.
Might be removing our .htaccess rules, removing our rules and deleting the
cache (so at least it wouldn't pull cached data), or just deleting the
cache. Quite frankly, my fear is if we're over-aggressive in .htaccess
clean-up, we could do even more harm. I'm sorry for the extra hassle.

**Comment by liquidsunshine** [comment]
Understood, and no worries.  I know it's not an easy problem to solve -- just wanted to add my story for context.  :)

**Comment by JamesGeiger** [comment]
Any news on this?

**Comment by chaselivingston** [comment]
Also requested here: https://twitter.com/BenjaminBrooks/status/866840700410712070
